runtime/peaq-dev/Cargo.toml

```diff
@@ -214,10 +214,15 @@ version = '0.0.2'
 
 [dependencies.peaq-pallet-storage]
 git = 'https://github.com/peaqnetwork/peaq-storage-pallet.git'
+branch = 'feat/1203500610868804_rpc-storage-pallet'
 default-features = false
-branch = 'dev'
 version = '0.1.1'
 
+[dependencies.peaq-pallet-storage-runtime-api]
+git = 'https://github.com/peaqnetwork/peaq-storage-pallet.git'
+branch = 'feat/1203500610868804_rpc-storage-pallet'
```

Comment:
After we merge the MR in the peaq-storage-pallet, please change the branch here because we'll delete the feature branch.